packages/p2p-v2/src/pages/my-profile/screens/MyProfileBalance/MyProfileBalance.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { useMemo, useState } from 'react';
-import { Text } from '@deriv-com/ui/dist/components/Text';
+import { useActiveAccount } from '@deriv/api';
+import { Text } from '@deriv-com/ui';
 import { AvailableP2PBalanceModal, DailyLimitModal } from '../../../../components';
 import { useAdvertiserStats, useDevice } from '../../../../hooks';
 import InfoOutlineIcon from '../../../../public/ic-info-outline.svg';
 import { numberToCurrencyText } from '../../../../utils';
-import './MyProfileBalance.scss';
 import { MyProfileDailyLimit } from '../MyProfileDailyLimit';
-import { useActiveAccount } from '@deriv/api';
+import './MyProfileBalance.scss';
```

Comment:
this import is removed but seems like the file is not deleted...?

Reply:
Its just sorted